_includes/macros/activity-snippets.html

```diff
@@ -0,0 +1,111 @@
+
+
+{# ==========================================================================
+
+   activity_snippets()
+
+   ==========================================================================
+
+   Description:
+
+   List a bunch of activity snippets when given:
+
+   tag:            A tag to use in the query.
+
+   activity_types: An array of activity types.
+                   Possible values: posts, op-ed, press release, speech, testimony
+
+   ========================================================================== #}
+
+{% macro render(tag, activity_types=['posts', 'op-ed', 'press release', 'speech', 'testimony', 'cfpb_report']) %}
```

Comment:
@kurtw please scrutinize the addition of `cfpb_report`. Is that how it needs to be?